onnxruntime/test/optimizer/resnet50_fusion_test.cc

```diff
@@ -27,9 +28,9 @@ class ResNet50FusionTests : public ::testing::Test {
       GTEST_SKIP() << "Failed to load model: " << fp16_model_path;
     }
   }
-  std::basic_string<ORTCHAR_T>  fp32_model_path = ORT_TSTR("../models/opset10/Resnet50_Fusion_Testing/resnet50.onnx");
+  std::basic_string<ORTCHAR_T>  fp32_model_path = ORT_TSTR("../models/opset10/Resnet50_Fusion_Testing_fp16/resnet50.onnx");
```

Comment:
No, Keep this line unchanged. I put two onnx files are in two different folders.